CHANGELOG.md

```diff
@@ -4,109 +4,163 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.6.1]
```

Comment:
i think you should revert the changelog, turn off format on save, then recommit

Reply:
Yes, forgot to turn off prettier